File: backend/src/main/java/com/schoolfinder/SchoolFinderApplication.java
package com.schoolfinder;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class SchoolFinderApplication {
    public static void main(String[] args) {
        SpringApplication.run(SchoolFinderApplication.class, args);
    }
}

File: backend/src/main/kotlin/com/schoolfinder/SchoolFinderApplication.kt
package com.schoolfinder

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SchoolFinderApplication

fun main(args: Array<String>) {
    runApplication<SchoolFinderApplication>(*args)
}
